packages/react/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src/**/*", "tests/**/*"]
+  "compilerOptions": {
+    "jsx": "react",
+  },
+  "include": ["src/**/*"]
```

Comment:
just one question: why did you remove tests from here?

Reply:
As the project is using Jest along with ts-jest to transform the TypeScript files which gives Jest allows typechecking to work.
Without ts-jest then you wouldn't get typechecking when running the tests as Jest uses Babel for TypeScript transforming.

But we should probably add `"test/**/*"` back in the `include` array for both `ts.config.json` in the packages folders incase you want to use TypeScript directly via the cli `yarn typecheck`.

Reply:
@andykenward gotcha. so I'll wait for you to add it back before merging. Cheers!

Reply:
@peduarte Done.

But if the project ever changes the build setup it may need removing.